internal/watcher/watcher.go

```diff
@@ -200,45 +210,19 @@ func (d Differential) WatchFn(path string) error {
 	}
 
 	if info.Size() >= MaxFileSize {
-		log.Printf("File equal or higher than %v MB", MaxFileSize/1024/1024)
+		log.Printf("File equal or higher than %v MB", MaxFileSize/sizeMb)
 
 		srcFile, ok := d.previousContents[path]
 		filePath := after(string(srcFile), "FilePath##")
 		containFilePath := strings.Contains(string(srcFile), "FilePath##")
 
 		if ok && !containFilePath {
 			// After watched file updated, the size higher than threshold
-
-			err = CopyFile(path, TemporaryDirectory+"/"+info.Name())
-			if err != nil {
-				return fmt.Errorf("error copy file  %w", err)
-			}
-
-			// find diff
-			contents, err := ioutil.ReadFile(path)
-			if err != nil {
-				return fmt.Errorf("ioutil.ReadFile: %w", err)
-			}
-
-			checksum := md5.Sum(contents) //nolint:gosec
-			if checksum == d.previousChecksum[path] {
-				return nil
-			}
-
-			diff := cmp.Diff(string(d.previousContents[path]), string(contents))
-
-			// set new previous content
-			contentName := "FilePath##" + TemporaryDirectory + "/" + info.Name()
-			d.previousContents[path] = []byte(contentName)
-
-			if err := d.sendDiff(path, diff); err != nil {
-				return fmt.Errorf("d.sendDiff: %w", err)
-			}
-
-			log.Printf("Sent the diff (size = %d)", len(diff))
-
+			d.WatchLargeFileFn(info, path)
```

Comment:
This function returns an error.  If error is not nil, we will need to handle it after calling the function.